release/src/Main.hs

```diff
@@ -28,7 +29,8 @@ main = do
       liftIO $ createDirIfMissing True releaseDir
       $logInfo "Checking if we should release"
       sdkVersion <- readVersionAt "HEAD"
-      release <- isReleaseCommit
+      -- release <- isReleaseCommit
+      let release = True
```

Comment:
This definitely needs to go back before we merge.

Reply:
Indeed! Should never of got commited!

Reply:
Fixed in 7a8baf12b